hatchery/hatchery.go

```diff
@@ -306,6 +330,7 @@ func launch(w http.ResponseWriter, r *http.Request) {
 		Config.Logger.Printf("Debug: Nextflow is not enabled: skipping Nextflow resources creation")
 	}
 
+	allpaymodels, err := getPayModelsForUser(userName)
 	if err != nil {
```

Comment:
do you need this above the new line you added as well to catch errs from before?

Reply:
i don't think so, i just moved the `allpaymodels` var initialization to right above where it's being used